src/modal/modal-window.ts

```diff
@@ -41,8 +44,8 @@ export class NgbModalWindow implements OnInit,
 
   constructor(private _elRef: ElementRef, private _renderer: Renderer) {}
 
-  backdropClick(): void {
-    if (this.backdrop === true) {
+  backdropClick($event): void {
+    if (this.backdrop === true && !this._isNodeChildOfAnother(this.contentEl.nativeElement, $event.target)) {
```

Comment:
The block
```ts
this._isNodeChildOfAnother(this.contentEl.nativeElement, $event.target)
```
is longer than how one would do it natively, i.e.
```ts
this.contentEl.nativeElement.contains($event.target);
```
Maybe this should be changed to a shorter name like `isContainedBy`, or just use the native version.